src/brain/issueLabelHandler/followups.ts

```diff
@@ -102,16 +103,18 @@ export async function updateCommunityFollowups({
   });
 
   const itemId: string = await addIssueToGlobalIssuesProject(
+    app,
     payload.issue.node_id,
     repo,
     issueNumber,
     octokit
   );
 
   await modifyProjectIssueField(
+    app,
     itemId,
     WAITING_FOR_PRODUCT_OWNER_LABEL,
-    STATUS_FIELD_ID,
+    app.project.status_field_id,
```

Comment:
Here's another big thing throughout, taking config from the `app` vs. the global config vars.